Dockerfile

```diff
@@ -3,8 +3,8 @@ FROM debian:jessie
 MAINTAINER Chris Kuehl <ckuehl@yelp.com>
 RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
     apt-get install -y --no-install-recommends \
-        build-essential devscripts equivs procps psmisc libpcre3 && \
+        build-essential devscripts equivs && \
     apt-get clean
 WORKDIR /mnt
 
-ENTRYPOINT mk-build-deps -i && make builddeb
+ENTRYPOINT mk-build-deps -i --tool 'apt-get --no-install-recommends -y' && make builddeb
```

Comment:
The default is `--tool 'apt-get --no-install-recommends`, we need to add `-y` so it is noninteractive.